README.md

```diff
@@ -1,30 +1,69 @@
-# 👾 processbot
-
-# Commands
-
-To be posted as a comment in the Pull Request. **Your whole comment should only
-have the command**.
+# Introduction
+
+parity-processbot is a
+[GitHub App](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps)
+which drives the
+[Companion Build System](https://github.com/paritytech/parity-processbot/issues/327)'s
+merge process. It receives [commands](#commands) and from subsequent GitHub
+events it will resume processing all pending pull requests until they can be
+merged; a more detailed breakdown is available in the
+[Implementation section](#implementation).
+
+Note that parity-processbot does not implement the Companion Build System's
+cross-repository integration check, which is done on CI (see
+[check_dependent_project](https://github.com/paritytech/pipeline-scripts#check_dependent_project)
```

Comment:
This link is broken at the moment because https://github.com/paritytech/pipeline-scripts/pull/26 is not merged yet